README.md

```diff
@@ -14,37 +14,77 @@ Borsh stands for _Binary Object Representation Serializer for Hashing_. It is me
 safety, speed, and comes with a strict specification.
 
 ## Examples
-### Serializing an object
+
+### (De)serializing a Value
 ```javascript
-const value = new Test({ x: 255, y: 20, z: '123', q: [1, 2, 3] });
-const schema = new Map([[Test, { kind: 'struct', fields: [['x', 'u8'], ['y', 'u64'], ['z', 'string'], ['q', [3]]] }]]);
-const buffer = borsh.serialize(schema, value);
+import * as borsh from 'borsh';
+
+const encodedU16 = borsh.serialize('u16', 2);
+const decodedU16 = borsh.deserialize('u16', encodedU16);
+
+const encodedStr = borsh.serialize('string', 'testing');
+const decodedStr = borsh.deserialize('string', encodedStr);
 ```
 
-### Deserializing an object
+### (De)serializing an Object
 ```javascript
-const newValue = borsh.deserialize(schema, Test, buffer);
+import * as borsh from 'borsh';
+import BN from 'bn.js';
+
+const value = {x: 255, y: new BN(20), z: '123', arr: [1, 2, 3]};
```

Comment:
Since bn is dropped this example might not work, let's use `20n` or `BigInt(20)`